cernopendata_client/downloader.py

```diff
@@ -61,8 +63,9 @@ def get_download_files_by_name(names=None, file_locations=None):
 
 def get_download_files_by_regexp(regexp=None, file_locations=None, filtered_files=None):
     """
-    Kwarg filtered_files - List of file locations filtered by previous filters(if any)
-    Returns the list of files filtered by a regular expression
+    Kwarg filtered_files - List of file locations filtered by previous filters(if any).
```

Comment:
We don't have kwarg anymore...

Reply:
Removed `Kwarg` 👍

Reply:
Please also invert the docstring, it should start with one headline, and details about parameters later, e.g.

```python
def get_current_commit(srcdir):
    """Return information about git commit checked out in the given directory.

    :param srcdir: source code directory
    :type srcdir: str

    :return: commit information composed of brief SHA1 and subject
    :rtype: str
    """
    pass
```

We don't have to introduce PEP247 param/type at this stage, but let's at least put headline before the details...